### PR TITLE
router and top level pages

### PR DESCRIPTION
**Pull request by cupcakecoders**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,23 +1,17 @@
 import React from 'react';
-import logo from './logo.svg';
+//import logo from './logo.svg';
```
Comment (MichaelWalker) [inline src/App.tsx]: we can probably just delete this line of code instead of commenting it out.

File: src/App.test.tsx [hunk 2]
```diff
@@ -2,8 +2,6 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('renders app.js', () => {
   const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
```
Comment (MichaelWalker) [inline src/App.test.tsx:8]: Looks like we've really just deleted the test.
Can we instead test for an element that does still exist?

File: src/Router.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+﻿import React from "react";
+// @ts-ignore
```
Comment (MichaelWalker) [inline src/Router.tsx]: As discussed - looks like we haven't got the types working properly! I think we shouldn't need this `ts-ignore`

**Review by MichaelWalker — CHANGES_REQUESTED** [review]
Couple of tiny tweaks, but otherwise looks good.

File: src/App.tsx (hunk 1 above)
Comment (cupcakecoders) [reply src/App.tsx]: Deleted reference to logo.

File: src/Router.tsx (hunk 3 above)
Comment (cupcakecoders) [reply src/Router.tsx]: deleted ts-ignore

File: src/App.test.tsx (hunk 2 above)
Comment (cupcakecoders) [reply src/App.test.tsx:8]: Lol! I've made a test to check the url for the curiosity page.

**Review by MichaelWalker — APPROVED** [review]

**Pull request merged**